virt/live_migration/virt-about-live-migration.adoc

```diff
@@ -0,0 +1,42 @@
+:_content-type: ASSEMBLY
+[id="virt-about-live-migration"]
+= About live migration
+include::_attributes/common-attributes.adoc[]
+:context: virt-about-live-migration
+
+toc::[]
+
+Live migration is the process of moving a running virtual machine (VM) to another node in the cluster without interrupting the virtual workload or access. By default, live migration traffic is encrypted using Transport Layer Security (TLS).
+
+You can manually xref:../../virt/live_migration/virt-initiating-live-migration.adoc#virt-initiating-live-migration[initate live migration] for a specific VM.
+
+You can monitor the progress of all live migrations on the xref:../../virt/getting_started/virt-web-console-overview.adoc#overview-migrations_virt-web-console-overview[*Overview* -> *Migrations* tab] in the web console.
+
+You can view the metrics of a VM migration on the xref:../../virt/getting_started/virt-web-console-overview.adoc#virtualmachine-details-metrics_virt-web-console-overview[*VirtualMachine details* -> *Metrics* tab] in the web console.
+
+.Prerequisites
+
+* The cluster must have shared storage with `ReadWriteMany` (RWX) access mode.
+* The cluster must have sufficient RAM and network bandwidth.
++
+[NOTE]
+====
+You must ensure that there is enough memory request capacity in the cluster to support node drains that result in live migrations. You can determine the approximate required spare memory by using the following calculation:
+
+----
+Product of (Maximum number of nodes that can drain in parallel) and (Highest total VM memory request allocations across nodes)
+----
+
+The default xref:../../virt/live_migration/virt-configuring-live-migration#virt-configuring-live-migration-limits_virt-configuring-live-migration[number of migrations that can run in parallel] in the cluster is 5.
+====
+
+* If a VM uses a host model CPU, the nodes must support the CPU.
+* A xref:../../virt/vm_networking/virt-dedicated-network-live-migration.adoc#virt-dedicated-network-live-migration[dedicated Multus network] for live migration is highly recommended. A dedicated network minimizes the effects of network saturation on tenant workloads during migration.
+
+* A VM created from a custom template must be configured with a xref:../../virt/live_migration/virt-configuring-live-migration.adoc#virt-configuring-vm-eviction-strategy-cli_virt-configuring-live-migration[`LiveMigrate` eviction strategy].
```

Comment:
I think this one is not correct: in 4.14 we have a global evictionStrategy configuration which is `LiveMigrate` by default, so no need to add it to the VM created from template.

Reply:
I will remove this.